src/show.jl

```diff
@@ -22,14 +22,31 @@ Base.show(io::IO, p::TypesWithShow) = show(io, MIME"text/plain"(), p)
 # VARIABLES
 function _show(io::IO, mime::MIME, var::AbstractVariable)
     base, indices = name_base_indices(var)
-    if isempty(indices)
-        print(io, base)
+    if isconj(var)
+        for c in base
+            print(io, c, '\u0305')
+        end
     else
         print(io, base)
+    end
+    if !isempty(indices)
         print_subscript(io, mime, indices)
     end
+    isrealpart(var) && print(io, "ᵣ")
+    isimagpart(var) && print(io, "ᵢ")
+end
+
+function _show(io::IO, mime::MIME"text/print", var::AbstractVariable)
+    if isconj(var)
+        for c in name(var)
+            print(io, c, '\u0305')
+        end
+    else
+        print(io, name(var))
+    end
+    isrealpart(var) && print(io, "ᵣ")
```

Comment:
Same here, what is this doing ?

Reply:
and the real/imaginary parts get an "r" or "i" subscript, `z_r` or `z_i`